english/test-driven-development-rules/AccordingToUncleBob.md

```diff
@@ -0,0 +1,8 @@
+Over the years I have come to describe Test Driven Development in terms of three simple rules. They are:
+
+
+1) You are not allowed to write any production code unless it is to make a failing unit test pass.
+
+2) You are not allowed to write any more of a unit test than is sufficient to fail; and compilation failures are failures.

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.